pkg/util/experimental.go

```diff
@@ -1,21 +1,8 @@
 package util
 
-import (
-	"github.com/go-kit/kit/log/level"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-)
-
-var experimentalFeaturesInUse = promauto.NewCounter(
-	prometheus.CounterOpts{
-		Namespace: "cortex",
-		Name:      "experimental_features_in_use_total",
-		Help:      "The number of experimental features in use.",
-	},
-)
+import "github.com/cortexproject/cortex/pkg/util/logutil"
 
 // WarnExperimentalUse logs a warning and increments the experimental features metric.
 func WarnExperimentalUse(feature string) {
```

Comment:
Alternative:

```
var WarnExperimentalUse = logutil.WarnExperimentalUse
```

I don't think we even need this. Let's just update the references.

Reply:
hahahaha good point. I  completely forgot about this.